src/modules/shared/notifications/GlobalNotificationsRoot.spec.tsx

```diff
@@ -0,0 +1,31 @@
+import { render, screen, fireEvent } from "@testing-library/react";
+import { GlobalNotificationsRoot } from "./GlobalNotificationsRoot";
+import { useNotify } from "./hooks/useNotify";
+import type { INotificationProps } from "@zendeskgarden/react-notifications";
+
+const payload = {
+  title: "Integration Title",
+  message: "Integration Message",
+  type: "info" as INotificationProps["type"],
+};
+
+function TestTrigger() {
+  const notify = useNotify();
+  return <button data-testid="trigger" onClick={() => notify(payload)} />;
+}
+
+describe("GlobalNotificationsRoot", () => {
```

Comment:
Nit: I feel we are testing the same thing in too many places. For example, we already tested this scenario in the `GlobalNotifications` component.

This component doesn't do anything special other than wrapping two other components and doesn't have any special logic.

I would personally not test this thing here, but it's also ok to have it here as an integration test at a higher level. Just pointing out that we don't need to test every single component if there is no value in it